src/main/java/io/debezium/connector/vitess/VitessConnector.java

```diff
@@ -150,7 +150,7 @@ public List<Map<String, String>> taskConfigs(int maxTasks, List<String> shards)
                     }
                     if (gtidStr == null) {
                         LOGGER.warn("No previous gtid found either for shard: {}, fallback to current", shard);
```

Comment:
Can you change the order of line 152 and 153, so you can print out the gtidStr value in the LOGGER.warn?

Reply:
Thanks! I overlooked it.
Added [5ea8780](https://github.com/debezium/debezium-connector-vitess/pull/112/commits/5ea878007cb93f43d5bf8ec74bb0193def13eff0).

Reply:
Thanks, looks good to me.